ts/features/itwallet/common/utils/itwMocksUtils.ts

```diff
@@ -4,7 +4,7 @@ import mdl from "../../__mocks__/mdl.json";
 import statusAttestation from "../../__mocks__/statusAttestation.json";
 import ts from "../../__mocks__/ts.json";
 import presentationClaims from "../../__mocks__/presentationClaims.json";
-import { type DisclosureClaim } from "../components/ItwClaimsDisclosure";
+import { type DisclosureClaim } from "../../issuance/components/ItwRequestedClaimsList";
```

Comment:
This is a component used for both issuance and remote presentation. I think it's better to leave it here in `common` folder. What do you think?

Reply:
Right, addressed in f85ca4a15844d4020107bbf361269fa8e7403682